kotlin-spark-api.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Kotlin for Apache® Spark™",
+  "properties": {
+    "spark": "3.2",
+    "version": "1.0.4"
+  },
+  "link": "https://github.com/JetBrains/kotlin-spark-api",
+  "dependencies": [
+    "org.jetbrains.kotlinx.spark:kotlin-spark-api-$spark:$version"
```

Comment:
Don't we need spark core/sql in dependencies, does it work without it?

Reply:
The dependencies are supplied inside Integration.kt. It seems to work when I try it manually with `%use file[...]`

Reply:
Cool then)